Makefile

```diff
@@ -68,6 +69,12 @@ modules: FORCE
 runtime: FORCE
 	cd runtime && $(MAKE)
 
+llvm-runtime-maybe: FORCE
```

Comment:
Coming to the party late... What do you think about try-llvm-runtime as a name for this target? I think that makes it more clear what is going on inside the recipe, plus it is more inline with the other speculative targets, like third-party-try-re2, third-party-try-opt, etc.